### PR TITLE
Advise against dates with decimal points

### PR DESCRIPTION
**Pull request by nik9000**
We accept dates with a decimal point like `2113413.13241324` and parse
them *somehow*. But there are cases where we'll lose precision on those
dates, see #70085. This advises folks not to use that format. We'll
continue to accept those dates for backwards compatibility but you
should avoid using them.

### CODE REVIEW

File: docs/reference/mapping/types/date.asciidoc [hunk 1]
```diff
@@ -7,11 +7,10 @@
 JSON doesn't have a date data type, so dates in Elasticsearch can either be:
 
 * strings containing formatted dates, e.g. `"2015-01-01"` or `"2015/01/01 12:10:30"`.
-* a long number representing _milliseconds-since-the-epoch_.
-* an integer representing _seconds-since-the-epoch_.
+* a number representing _milliseconds-since-the-epoch_.
```
[the remark this answers is not in the document]
Comment (jpountz) [reply docs/reference/mapping/types/date.asciidoc:10]: By using the `epoch_second` format I would assume?

[the remark this answers is not in the document]
Comment (nik9000) [reply docs/reference/mapping/types/date.asciidoc:10]: Oh yeah! I'd not had to do that in a while. I'll restore this an document how to do it.

**Comment by nik9000** [comment]
@jpountz, I added some documentation for using `epoch_second`.

File: docs/reference/mapping/types/date.asciidoc [hunk 2]
```diff
@@ -139,3 +153,55 @@ The following parameters are accepted by `date` fields:
 <<mapping-field-meta,`meta`>>::
 
     Metadata about the field.
+
+
+[[date-epoch-seconds]]
+==== Epoch seconds
+
+If you need to send dates as _seconds-since-the-epoch_ then make sure the
+`format` lists `epoch_seconds`:
```
Comment (jpountz) [inline docs/reference/mapping/types/date.asciidoc:162]:
```suggestion
`format` lists `epoch_second`:
```

**Review by jpountz — APPROVED** [review]

**Pull request merged**